README.md

```diff
@@ -160,8 +160,7 @@ is valuable.
 ## Google Summer of Code
 
 We participated in
-[GSoC](https://developers.google.com/open-source/gsoc/) last year and
-hope to do so again in 2017.  For guidance, please read
+[GSoC](https://developers.google.com/open-source/gsoc/) last year and participating in 2017 too.  For guidance, please read
```

Comment:
Yea, leaving last year is fine for this year. It'll have to be changed next year, assuming Zulip happens during 2018.